packages/@lwc/synthetic-shadow/src/shared/node-ownership.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import { defineProperty, isNull, isUndefined } from '@lwc/shared';
+import { parentNodeGetter } from '../env/node';
+
+// DO NOT CHANGE this:
+// these two values need to be in sync with engine
+const OwnerKey = '$$OwnerKey$$';
```

Comment:
For future reference, please add a comment here for each key type.
```suggestion
// Used as a back reference to identify the host element(owner) of a dom node
const OwnerKey = '$$OwnerKey$$';
```